### ISSUE TITLE
[4.0]quickicon color contrast (.j-links-link)

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#428]
![image](https://user-images.githubusercontent.com/1296369/63165347-1332de80-c023-11e9-9f43-a81474b04a4a.png)

#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Control Panel - New Template - Administration](http://localhost/joomla-cms/administrator/index.php#ui-skip-40)
#### Element path
#plg_quickicon_extensionupdate > .quickicon-text.align-items-center.d-flex > .j-links-link
#### Snippet
`<span class="j-links-link">Updates are available! <span class="badge badge-light">1</span></span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.48 (foreground color: #495057, background color: #f9acaa, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 76.0.3809.100
This accessibility issue was found using Accessibility Insights for Web 2.8.0 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
